draft-ietf-tsvwg-multipath-dccp.mkd

```diff
@@ -847,21 +847,24 @@ address becomes invalid (e.g., if the interface disappears), the
 affected host SHOULD announce this so that the peer can remove subflows
 related to this address.
 
-This is achieved through the Remove Address (REMOVE_ADDR) option which
+This is achieved through the Remove Address (MP_REMOVEADDR) option which
 will remove a previously added address (or list of addresses) from a
 connection and terminate any subflows currently using that address.
 
-For security purposes, if a host receives a REMOVE_ADDR option, it must
+For security purposes, if a host receives a MP_REMOVEADDR option, it must
 ensure the affected path(s) are no longer in use before it instigates
-closure. Typical DCCP validity tests on the subflow (e.g., packet type
-specific sequence and acknowledgement number check) MUST also be
-undertaken. An implementation can use indications of these test failures
-as part of intrusion detection or error logging.
-
-The sending and receipt of this message SHOULD trigger the sending of
-DCCP-Close and DCCP-Reset by client and server, respectively on the
-affected subflow(s) (if possible), as a courtesy to cleaning up
-middlebox state, before cleaning up any local state.
+closure. The receipt of MP_REMOVEADDR SHOULD first trigger
+the sending of a DCCP keepalive on the path, and if a response is
```

Comment:
The keepalive procedure is copied from MPTCPv0 RFC. From my view this makes sense. What couold be an equivalent in DCCP, as RFC4340 does not provide a keepalive functionality.